specutils/analysis/moment.py

```diff
@@ -0,0 +1,78 @@
+"""
+A module for analysis tools focused on determining the moment of
+spectral features.
+"""
+
+import numpy as np
+from ..manipulation import extract_region
+from .utils import computation_wrapper
+
+
+__all__ = ['moment']
+
+
+def moment(spectrum, regions=None, order=0, axis=-1):
+    """
+    Estimate the moment of the spectrum.
+
+
+    Parameters
+    ----------
+    spectrum : `~specutils.spectra.spectrum1d.Spectrum1D`
```

Comment:
Will this also work on `SpectrumCollection`? *in principle* it should, and maybe we should add a test if it does (and if not make an issue to fix it later?)

Reply:
I verified now, it does not work with `SpectrumCollection`. I created issue #772 .

Reply:
Ok, looks good - we can deal with it later then.  Will merge!